src/Core/Geographic/Coordinates.js

```diff
@@ -148,8 +150,23 @@ function _convert(coordsIn, newCrs, target) {
         }
 
         if (coordsIn.crs in proj4.defs && newCrs in proj4.defs) {
-            const p = instanceProj4(coordsIn.crs, newCrs).forward([coordsIn._values[0], coordsIn._values[1]]);
-            return target.set(newCrs, p[0], p[1], coordsIn._values[2]);
+            // little hack due to our choice to use Radian for the EPSG:4326 projection (instead of the standard EPSG:4326 in Degree)
+            const needToHack = coordsIn.crs === 'EPSG:4326' && coordsIn._internalStorageUnit == UNIT.RADIAN;
+            const val0 = needToHack ? mE.radToDeg(coordsIn._values[0]) : coordsIn._values[0];
+            const val1 = needToHack ? mE.radToDeg(coordsIn._values[1]) : coordsIn._values[1];
+
+            // there is a bug for converting anything to 4978 with proj4
```

Comment:
Do converting from 4978 works correctly? The linked issue seems to indicate that both projections `to` and `from` 4978 aren't supported.

Reply:
I don't know, I don't have an example the test this case.

Reply:
Ok, I tried and converting From 4978 doesn't work correctly.